specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -4298,6 +4298,42 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}": {
```

Comment:
>locations [](start = 68, length = 9)

as per breaking change policy, adding new paths to existing api version is not allowed - please check the results of the CI validations that have run on your PR

Reply:
Sure, this is for proposal review. I will use new API version once this looks good. Thanks for reminding!